packages/edit-site/src/components/editor-canvas-container/index.js

```diff
@@ -46,6 +48,7 @@ const {
 
 function EditorCanvasContainer( {
 	children,
+	showCloseButton = false,
```

Comment:
How about instead of a new prop we don't show the button if `onClose` is not provided?